src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -461,6 +467,32 @@ private void verifyUpload(Exporter3Configuration ex3Config, String uploadId, Str
 
         long participantModifiedOn = Long.parseLong(rowMap.get("modifiedOn"));
         assertTrue(participantModifiedOn > oneHourAgo.getMillis());
+
+        // Verify the record in Bridge.
+        ForWorkersApi workersApi = adminDeveloperWorker.getClient(ForWorkersApi.class);
+        HealthDataRecordEx3 record = workersApi.getRecordEx3(IntegTestUtils.TEST_APP_ID, uploadId).execute().body();
+        assertTrue(record.isExported());
+        assertTrue(record.getExportedOn().isAfter(oneHourAgo));
```

Comment:
I think we can move this and everything after this to testUpload(). verifyUpload() is called twice because the upload is exported to Synapse twice (once for the app, once for the study), but there's only one record, so verifying it twice doesn't make sense.